SECURITY.md

```diff
@@ -0,0 +1,52 @@
+# Security Components in EdgeX Foundry Services
+
+Starting from Fuji release, EdgeX includes enhanced security features. There are 3 major components that are responsible for security
+features: 
+
+- [Security-secrets-setup] (https://github.com/edgexfoundry/edgex-go/blob/master/cmd/security-secrets-setup/README.md)
+- [Security-secretstore-setup] (https://github.com/edgexfoundry/edgex-go/blob/master/cmd/security-secretstore-setup/README.md)
+- [Security-proxy-setup] (https://github.com/edgexfoundry/edgex-go/blob/master/cmd/security-proxy-setup/README.md)
```

Comment:
The space between the right bracket and left paren is causing this to render incorrectly.  Also, would a relative URL to README.md work?

Reply:
Fixed.